Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:14-alpine
+
+WORKDIR /app
+
+COPY ./package.json .
```

Comment:
Do we ignore the existing yarn.lock?

Reply:
I think we probably dont need the yarn.lock, as long as we have the node_modules it should be fine. The yarn lock just tells yarn where to download the dependencies from and how to resolve conflicting versions